src/Carbon/Lang/so.php

```diff
@@ -10,6 +10,50 @@
  */
 
 /*
- * Unknown default region, use the first alphabetically.
+ * Author:
+ * - Abdifatah Abdilahi(@abdifatahz)
  */
-return require __DIR__.'/so_DJ.php';
+return [
+    'year' => '{1}:count sanad|{0}:count sanadood|]1,Inf[:count sanadood',
+    'a_year' => '{1}a sanad|{0}:count sanadood|]1,Inf[:count sanadood',
+    'y' => '{1}:countsn|{0}:countsns|]1,Inf[:countsn',
+    'month' => '{1}:count bil|{0}:count bilood|]1,Inf[:count bilood',
+    'a_month' => '{1}a bil|{0}:count bilood|]1,Inf[:count bilood',
+    'm' => '{1}:countbil|{0}:countbil|]1,Inf[:countbil',
+    'week' => '{1}:count isbuuc|{0}:count isbuuc|]1,Inf[:count isbuuc',
+    'a_week' => '{1}a week|{0}:count isbuuc|]1,Inf[:count isbuuc',
+    'w' => ':countis',
+    'day' => '{1}:count maalin|{0}:count maalmood|]1,Inf[:count maalmood',
+    'a_day' => '{1}maalin|{0}:count maalmood|]1,Inf[:count maalmood',
+    'd' => ':countml',
+    'hour' => '{1}:count saac|{0}:count saac|]1,Inf[:count saac',
```

Comment:
When the translation is the same for any number, you don't need to use the pluralization syntax:
```php
'hour' => ':count saac',
```